pkg/operator/kubeletversionskewcontroller/kubelet_version_skew_controller.go

```diff
@@ -0,0 +1,258 @@
+package kubeletversionskewcontroller
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/blang/semver"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/management"
+	"github.com/openshift/library-go/pkg/operator/status"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/runtime"
+	corev1listers "k8s.io/client-go/listers/core/v1"
+)
+
+const (
+	KubeletMinorVersionUpgradeableConditionType = "KubeletMinorVersionUpgradeable"
+
+	KubeletVersionUnknownReason                     = "KubeletVersionUnknown"
+	KubeletMinorVersionSyncedReason                 = "KubeletMinorVersionsSynced"
+	KubeletMinorVersionSupportedNextUpgradeReason   = "KubeletMinorVersionSupportedNextUpgrade"
+	KubeletMinorVersionUnsupportedNextUpgradeReason = "KubeletMinorVersionUnsupportedNextUpgradeReason"
```

Comment:
nit: probably drop `Reason` in the value here (but keep it in the variable name)

Reply:
Done.